packages/affiliates/test/mocha/affiliates.js

```diff
@@ -21,129 +32,249 @@ const devRewardsToDistribute = "50000";
+      balanceHistories[0][1].handleEvent(0, {
+        name: "PositionCreated",
+        // creating a position for address "aa" with 2 collateral 1 synthetic
+        args: ["aa", "2", "1"],
+        blockTimestamp: 0
+      });
+      balanceHistories[0][1].finalize();
+      const result = affiliates.utils.calculateRewards(params);
+      assert.equal(result.empPayouts["a"], totalRewards);
     });
-  });
-  it("getCoingeckoPriceHistory", async function() {
-    this.timeout(10000);
-    const [, address] = collateralTokens;
-    const result = await affiliates.utils.getCoingeckoPriceHistory(address, "usd", startingTimestamp, endingTimestamp);
-    assert.ok(result.prices.length);
-  });
-  it("getSyntheticPriceHistory", async function() {
-    this.timeout(10000);
-    const [, address] = empContracts;
-    const result = await affiliates.utils.getSyntheticPriceHistory(address, startingTimestamp, endingTimestamp);
-    assert.ok(result.prices.length);
-  });
-  it("getBlocks", async function() {
-    this.timeout(30000);
-    const result = await affiliates.utils.getBlocks(startingTimestamp, startingTimestamp + 60 * 1000 * 5);
-    assert.ok(result.length);
-    const [first] = result;
-    assert(first.timestamp > 0);
-    assert(first.number > 0);
-  });
-  it("getEmpDeployerHistory", async function() {
-    this.timeout(10000);
-    const result = await affiliates.utils.getEmpDeployerHistory(empCreator, startingTimestamp);
-    assert(result.length);
-  });
-  it("calculateValue", async function() {
-    // these are all stable coins so they should roughly be around 1 dollar
-    // epsilon is high because variations could be nearly a dollar in any direction
-    const epsilon = 10n ** 17n;
-    const target = 10n ** 18n;
-    // btc example
-    let result = affiliates.utils.calculateValue(target, "15437.012543625458", "65019421301142", 8, 18).toString();
-    let diff = BigInt(result) - target;
-    assert(diff > 0 ? diff : -diff < epsilon);
-    // eth example
-    result = affiliates.utils.calculateValue(target, "452.1007409061987", "2201527860335072", 18, 18).toString();
-    diff = BigInt(result) - target;
-    assert(diff > 0 ? diff : -diff < epsilon);
-    // perlin example
-    result = affiliates.utils.calculateValue(target, "0.021647425848012932", "45829514207149404216", 18, 18).toString();
-    diff = BigInt(result) - target;
-    assert(diff > 0 ? diff : -diff < epsilon);
-  });
-  it("calculateValueFromUsd", async function() {
-    // these are all stable coins so they should roughly be around 1 dollar
-    // epsilon is high because variations could be nearly a dollar in any direction
-    const target = 10n ** 18n;
-    const syntheticPrice = 26.358177384415466;
-    let result = affiliates.utils.calculateValueFromUsd(target, 0, syntheticPrice, 18, 18).toString();
-    assert.equal(result, toWei(syntheticPrice.toFixed(18)));
+    it("should work with basic test cases", function() {
```

Comment:
nit: are these two test cases named the same thing?

Reply:
I have an update coming to better name these test cases, it was an oversight